save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -101,6 +101,25 @@ class VulnerabilityController(
         ResponseEntity.ok("Vulnerability was successfully updated")
     }
 
+    @DeleteMapping("/delete")
```

Comment:
Suggestion to use `Post` instead of `Delete` as in delete we might rely on request body, but it is not always expected: [DELETE doc](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE)